CHANGES.md

```diff
@@ -3,6 +3,8 @@ next
 
 - Add `Attribute.declare_with_name_loc` (#.., @diml)
 
+- Let the tool name pass throught when used as a -ppx (#..., @diml)
```

Comment:
Don't forget to update with the PR name (same two lines up).

Reply:
Indeed, thanks